lib/matplotlib/backends/backend_pdf.py

```diff
@@ -2423,8 +2425,11 @@ def __init__(self, filename, keep_empty=True):
         keep_empty: bool, optional
             If set to False, then empty pdf files will be deleted automatically
             when closed.
+        metadata: dictionary, optional
+            Information dictionary object (see PDF reference section 10.2.1
```

Comment:
The original parameters all don't have a space before the colon, and rendered fine, so I don't think that's a requirement.